### ISSUE TITLE
Compiling Errors: lib/jpeg.c needs update

### ISSUE THREAD

**Issue by SergeGardien** [matthiaskramm/swftools#37]
SWFTool doesn't compile under Mac OS X 10.11.6.
You need to update swftools/lib/jpeg.c as per advice of a jpeg developer (at [https://trac.macports.org/ticket/42735#comment:8](https://trac.macports.org/ticket/42735#comment:8)) by substituting the following lines:
```
#ifdef HAVE_JPEGLIB
#define HAVE_BOOLEAN
#include <jpeglib.h>
```
with:
```
#ifdef HAVE_JPEGLIB
#ifndef FALSE            /* in case these macros already exist */
#define FALSE   0        /* values of boolean */
#endif
#ifndef TRUE
#define TRUE    1
#endif
#define HAVE_BOOLEAN
#include <jpeglib.h>
```

**Comment by apjanke** [comment]
Ping. This is still an issue. Build is failing on macOS 10.13. (https://github.com/Homebrew/homebrew-core/issues/28559)

Are you interested in supporting macOS for this package?

**Comment by yoya** [comment]
I believe this issue has been resolved in PR #67 
Can I close this issue ?

**Comment by matthiaskramm** [comment]
Yeah, let's close.
